### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by pcw-mesa** [jethornton/7i96#8]
It looks like the python setuptools are missing from the install instructions
( at least its missing from my system = Mint 19.2 )

**Comment by jethornton** [comment]
I'll check that in the morning, are you running LinuxCNC on Mint 19.2?

**Comment by pcw-mesa** [comment]
On Wed, 20 Nov 2019, jethornton wrote:

> Date: Wed, 20 Nov 2019 15:45:54 -0800
> From: jethornton <notifications@github.com>
> Reply-To: jethornton/7i96
>     <reply@reply.github.com>
> To: jethornton/7i96 <7i96@noreply.github.com>
> Cc: pcw-mesa <pcw@mesanet.com>, Author <author@noreply.github.com>
> Subject: Re: [jethornton/7i96] Missing dependency (#8)
> 
> I'll check that in the morning, are you running LinuxCNC on Mint 19.2?

Yes. I updated my home machine (it was running Ubuntu 18.04) but the PC died I 
had planned to just to move the old hard drive to a new machine but it was a 
3.5" hard drive and the tiny new PC will only take a 2.5" So I just re-installed 
everything from scratch, starting with Mint 19.2. I did move my complete home 
directory (including Xilinx tools) from old drive to new so it was not too bad



Peter Wallace
Mesa Electronics

(\__/)
(='.'=) This is Bunny. Copy and paste bunny into your
(")_(") signature to help him gain world domination.
